src/opm/parser/eclipse/EclipseState/Schedule/Well/WellTestState.cpp

```diff
@@ -85,12 +100,25 @@ namespace Opm {
         return this->wells.size();
     }
 
-    std::vector<std::pair<std::string, WellTestConfig::Reason>> WellTestState::updateWell(const WellTestConfig& config, double sim_time) {
+    std::vector<std::pair<std::string, WellTestConfig::Reason>>
+    WellTestState::updateWell(const WellTestConfig& config,
```

Comment:
When you are here - maybe rename this function to the plural `updateWells` - since it clearly operates on a collection of wells.